docs/source/install/profile.rst

```diff
@@ -207,3 +207,20 @@ Queue: dvd_usr_prod (Nvidia P100 GPUs)
 
 .. literalinclude:: profiles/davide-cineca/gpu_picongpu.profile.example
    :language: bash
+
+JURECA (JSC)
+--------------------
+
+**System overview:** `link <http://www.fz-juelich.de/ias/jsc/EN/Expertise/Supercomputers/JURECA/JURECA_node.html>`_
+
+**User guide:** `link <http://www.fz-juelich.de/ias/jsc/EN/Expertise/Supercomputers/JURECA/UserInfo/UserInfo_node.html>`_
+
+**Production directory:** ``$SCRATCH`` (`link <http://www.fz-juelich.de/SharedDocs/FAQs/IAS/JSC/EN/JUST/FAQ_00_File_systems.html?nn=1297148>`_)
+
+For this profile to work, you need to download the :ref:`PIConGPU source code <install-dependencies-picongpu>` manually.
+
+Queue: batch (Intel Xeon E5-2680 v3 CPUs)
```

Comment:
Please add `2 x Intel ...`

Reply:
Sure. It's actually also really inconsistent between different machines in this document. I think the best formulation is [this one]( https://github.com/ComputationalRadiationPhysics/picongpu/blob/dev/docs/source/install/profile.rst#queue-defq-2x-intel-xeon-gold-6148-20-cores--20-hyperthreadscpu), also covering SMT. I guess I'll follow that and put `Queue: batch (2 x Intel Xeon E5-2680 v3 CPUs, 12 Cores + 12 HyperThreads per CPU)`.

Reply:
offline discussed: correct `... per node` is not required.